examples/demo/updater/updater_test.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package updater
+
+import (
+	"context"
+	"encoding/json"
+	"strconv"
+	"sync"
+	"testing"
+)
+
+func TestUpdater(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	latest := make(chan string)
+
+	base := New(ctx, func(b []byte) {
+		latest <- string(b)
+	})
+
+	child := NewNested(ctx, base.ForField("Foo"))
+
+	l := <-latest
+	if l != "{}" {
+		t.Errorf("Got %s, expected %s", l, "{}")
+	}
+
+	l = <-latest
+	if l != "{\"Foo\":{}}" {
+		t.Errorf("Got %s, expected %s", l, "{\"Foo\":{}}")
+	}
+
+	child.ForField("Bar")(interface{}((*int)(nil)))
+
+	l = <-latest
+	if l != "{\"Foo\":{\"Bar\":null}}" {
```

Comment:
use the assert (or require) library being used in the rest of the testing for Open Match?

Reply:
Stylistically, I prefer not to, and I haven't seen any guidance on this yet.  Is the assert library something we intentionally chose after reviewing options, or just something that someone chose at some point?